### DISCUSSION TITLE
Script to run again when hitting the back button

### DISCUSSION THREAD

**Discussion by kickbk** [gatsbyjs/gatsby#38100] [Help]
I am loading a script inside the body of a page with `<Script src="/scripts/contact.js" />`. I have `contact.js` in `static > scripts` with a simple `console.log("Hello")`.
When I navigate to the page containing the `<Script>`, it shows "Hello" in the console. I then click on a link to go to another page, and then on the back button to go back to the page that contains the `<Script>`. The script doesn't run again. I can't see anything in the console.
Is this expected? Is there a way to get the script to run after navigating into a page with the browser back button?

**Comment by kzsa** [comment]
https://react.dev/learn/synchronizing-with-effects
